+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Thu Dec 29 00:16:17 2022 - INFO  - Start environmental preparation.
+ DNF_INSTALL 'lxc-libs dnsmasq'
+ pkgs='lxc-libs dnsmasq'
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs 'lxc-libs dnsmasq' --node 1 --tempfile ''
+ tmpfile2=/tmp/tmp3rodfksj
+ '[' -z '' ']'
+ tmpfile=/tmp/tmp3rodfksj
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Thu Dec 29 00:17:17 2022 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Thu Dec 29 00:17:18 2022 - INFO  - Start testing...
+ test_execution lxc-net.service
+ service=lxc-net.service
++ date '+%Y-%m-%d %T'
+ log_time='2022-12-29 00:17:18'
+ test_restart lxc-net.service
+ service=lxc-net.service
+ systemctl restart lxc-net.service
Job for lxc-net.service failed because the control process exited with error code.
See "systemctl status lxc-net.service" and "journalctl -xeu lxc-net.service" for details.
+ CHECK_RESULT 1 0 0 'lxc-net.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='lxc-net.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'lxc-net.service restart failed'
+ LOG_ERROR 'lxc-net.service restart failed'
+ message='lxc-net.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lxc-net.service restart failed'
Thu Dec 29 00:17:19 2022 - ERROR - lxc-net.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Thu Dec 29 00:17:20 2022 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status lxc-net.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'lxc-net.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='lxc-net.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'lxc-net.service restart failed'
+ LOG_ERROR 'lxc-net.service restart failed'
+ message='lxc-net.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lxc-net.service restart failed'
Thu Dec 29 00:17:26 2022 - ERROR - lxc-net.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Thu Dec 29 00:17:27 2022 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop lxc-net.service
+ CHECK_RESULT 0 0 0 'lxc-net.service stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='lxc-net.service stop failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status lxc-net.service
+ grep 'Active: inactive'
+ CHECK_RESULT 1 0 0 'lxc-net.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='lxc-net.service stop failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'lxc-net.service stop failed'
+ LOG_ERROR 'lxc-net.service stop failed'
+ message='lxc-net.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lxc-net.service stop failed'
Thu Dec 29 00:17:33 2022 - ERROR - lxc-net.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Thu Dec 29 00:17:34 2022 - ERROR - ../common/common_lib.sh line 42
+ return 0
+ systemctl start lxc-net.service
Job for lxc-net.service failed because the control process exited with error code.
See "systemctl status lxc-net.service" and "journalctl -xeu lxc-net.service" for details.
+ CHECK_RESULT 1 0 0 'lxc-net.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='lxc-net.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'lxc-net.service start failed'
+ LOG_ERROR 'lxc-net.service start failed'
+ message='lxc-net.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lxc-net.service start failed'
Thu Dec 29 00:17:35 2022 - ERROR - lxc-net.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Thu Dec 29 00:17:36 2022 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status lxc-net.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'lxc-net.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='lxc-net.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'lxc-net.service start failed'
+ LOG_ERROR 'lxc-net.service start failed'
+ message='lxc-net.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'lxc-net.service start failed'
Thu Dec 29 00:17:42 2022 - ERROR - lxc-net.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Thu Dec 29 00:17:43 2022 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled lxc-net.service
+ service=lxc-net.service
++ systemctl is-enabled lxc-net.service
+ state=disabled
+ '[' disabled == enabled ']'
+ '[' disabled == disabled ']'
++ systemctl enable lxc-net.service
++ grep 'Created symlink'
++ awk '{print $3}'
+ symlink_file=/etc/systemd/system/multi-user.target.wants/lxc-net.service
+ find /etc/systemd/system/multi-user.target.wants/lxc-net.service
/etc/systemd/system/multi-user.target.wants/lxc-net.service
+ CHECK_RESULT 0 0 0 'lxc-net.service enable failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='lxc-net.service enable failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl disable lxc-net.service
Removed /etc/systemd/system/multi-user.target.wants/lxc-net.service.
+ find /etc/systemd/system/multi-user.target.wants/lxc-net.service
find: ‘/etc/systemd/system/multi-user.target.wants/lxc-net.service’: No such file or directory
+ CHECK_RESULT 1 0 1 'lxc-net.service disable failed'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='lxc-net.service disable failed'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ journalctl --since '2022-12-29 00:17:18' -u lxc-net.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
Dec 29 00:17:18 openEuler-riscv64 lxc-net[518]: Error: Unknown device type.
Dec 29 00:17:18 openEuler-riscv64 lxc-net[516]: Failed to setup lxc-net.
Dec 29 00:17:18 openEuler-riscv64 systemd[1]: lxc-net.service: Main process exited, code=exited, status=1/FAILURE
Dec 29 00:17:18 openEuler-riscv64 systemd[1]: lxc-net.service: Failed with result 'exit-code'.
Dec 29 00:17:18 openEuler-riscv64 systemd[1]: Failed to start LXC network bridge setup.
Dec 29 00:17:34 openEuler-riscv64 lxc-net[538]: Error: Unknown device type.
Dec 29 00:17:34 openEuler-riscv64 lxc-net[536]: Failed to setup lxc-net.
Dec 29 00:17:34 openEuler-riscv64 systemd[1]: lxc-net.service: Main process exited, code=exited, status=1/FAILURE
Dec 29 00:17:34 openEuler-riscv64 systemd[1]: lxc-net.service: Failed with result 'exit-code'.
Dec 29 00:17:34 openEuler-riscv64 systemd[1]: Failed to start LXC network bridge setup.
+ CHECK_RESULT 0 0 1 'There is an error message for the log of lxc-net.service'
+ actual_result=0
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of lxc-net.service'
+ '[' -z 0 ']'
+ '[' 1 -eq 0 ']'
+ test 0x == 0x
+ test -n 'There is an error message for the log of lxc-net.service'
+ LOG_ERROR 'There is an error message for the log of lxc-net.service'
+ message='There is an error message for the log of lxc-net.service'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'There is an error message for the log of lxc-net.service'
Thu Dec 29 00:17:52 2022 - ERROR - There is an error message for the log of lxc-net.service
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 28'
+ message='../common/common_lib.sh line 28'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 28'
Thu Dec 29 00:17:52 2022 - ERROR - ../common/common_lib.sh line 28
+ return 0
+ test_reload lxc-net.service
+ service=lxc-net.service
+ systemctl start lxc-net.service
Job for lxc-net.service failed because the control process exited with error code.
See "systemctl status lxc-net.service" and "journalctl -xeu lxc-net.service" for details.
+ systemctl reload lxc-net.service
+ grep 'Job type reload is not applicable'
Failed to reload lxc-net.service: Job type reload is not applicable for unit lxc-net.service.
+ CHECK_RESULT 0 0 0 'Job type reload is not applicable for unit lxc-net.service'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit lxc-net.service'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl status lxc-net.service
+ grep 'Active: active'
+ systemctl status lxc-net.service
+ grep 'inactive (dead)'
+ return 1
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Thu Dec 29 00:17:54 2022 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 6
+ test 6 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Dec 29 00:17:54 2022 - ERROR - failed to execute the case.
+ exit 6
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Thu Dec 29 00:17:55 2022 - INFO  - start environment cleanup.
+ systemctl stop lxc-net.service
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmp3rodfksj ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmp3rodfksj
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Thu Dec 29 00:18:12 2022 - INFO  - Finish environment cleanup!
